plugin-server/src/main/pluginsServer.ts

```diff
@@ -164,6 +167,19 @@ export async function startPluginsServer(
         })
 
         // use one extra Redis connection for pub-sub
+        const pubSubChannelsConfig: PubSubTaskMap =
```

Comment:
just fyi: not really familiar with this part/file & didn't look into it too much

Reply:
that's fine - this is the communication layer for django <> plugin server.

You'll notice we just kept the action stuff on the ingestion side (for now) and the `handleAlert` related stuff on the runner.